angular-mentoring/src/app/breadcrumbs/breadcrumbs.component.css

```diff
@@ -0,0 +1,9 @@
+.breadcrumbs__courses {
```

Comment:
[BEM](https://ru.bem.info) is a great methodology but to get as much profit as possible it should be used across all application. To achieve this `app.component.css` and `main.component.css` should be updated.